open-craft/edx-platform
# [SE-3329] Expose banner image URL in course API

Banner image URL is exposed on the course list and course details API
endpoints. The new `banner_image_url` API field has both the relative
and absolute URLs for the image it represents, if it is set.

**Screenshots**:
![99000525-b4b42300-2539-11eb-8379-6d219470e72b](https://user-images.githubusercontent.com/19173947/99500320-7b851400-297a-11eb-86f3-d4fe9b2846fe.png)
![99000523-b382f600-2539-11eb-84ea-61526843c73a](https://user-images.githubusercontent.com/19173947/99500326-7de76e00-297a-11eb-8fed-def31bfc9360.png)

**Sandbox URL**: N/A

**Testing instructions**: 

No need for testing, this is a cherry-picked commit to make sure we use the same code what we have on upstream.

**Author notes and concerns**:

After merging this PR, we need to make sure that the migrations applied correctly, because **the migration already exists in the DB, but with different DB column parameters**. 

**Reviewers**
